Resources/Prototypes/Catalog/Fills/Lockers/science.yml

```diff
@@ -13,3 +13,5 @@
       - id: NodeScanner
       - id: NetworkConfigurator
         prob: 0.5
+      - id: PlushieScientist
+        prob: 0.2
```

Comment:
I don't think adding this to lockers is a good idea in my opinion

Reply:
I found it interesting to find your department's plush toy in the locker at the beginning of the shift. This can be removed

Reply:
Let a maintainer discuss this one

Reply:
Don't bloat lockers with fluff, just what they need for their job.